### PR TITLE
#155450793 Fetch results of a collection

### PR DESCRIPTION
**Pull request by eric-elem**
## What does this PR do?
Fetches results of a collection from the database and sends them to the frontend.

## Tasks to be completed
- Add models for assertions and headers
- Add logic to save checks under a collection
- Add logic to run checks under a collection
- Add logic to retrieve and return results
- Add tests

## How can this be tested
- Run the application and visit the collections page and select a collection to view it's details. Add/Run checks under that collection and the results shall be displayed.

## What is the relevant pivotal tracker story?
https://www.pivotaltracker.com/n/projects/2152779/stories/155450793

## Screenshot
screen shot 2018-04-10 at 11 56 52

### CODE REVIEW

File: application/base_model.py [hunk 1]
```diff
@@ -29,7 +30,8 @@ def save(self):
             db.session.add(self)
             db.session.commit()
             return True
-        except SQLAlchemyError:
+        except SQLAlchemyError as e:
```
Comment (Ashaba) [inline application/base_model.py:33]: Remove print statements

**Review by Ashaba — COMMENTED** [review]
LGTM

File: application/base_model.py (hunk 1 above)
Comment (eric-elem) [reply application/base_model.py:33]: Removed. Thanks.

**Comment by muhallan** [comment]
LGTM

**Pull request merged**